workflows/epigenetics/average-bigwig-between-replicates/average-bigwig-between-replicates.ga

```diff
@@ -11,7 +11,7 @@
     "format-version": "0.1",
     "license": "MIT",
     "release": "0.2",
-    "name": "Average Bigwig between replicates",
+    "name": "BigWig Replicates Averaging Tool",
```

Comment:
```suggestion
    "name": "BigWig Replicates Averaging Workflow",
```
I am usually not a fan of specifying it's a workflow when that is obvious, but I think in this case it improves the name ?